crypto/hash/hmac_ossl.c

```diff
@@ -95,146 +88,90 @@ static srtp_err_status_t srtp_hmac_alloc (srtp_auth_t **a, int key_len, int out_
     (*a)->out_len = out_len;
     (*a)->key_len = key_len;
     (*a)->prefix_len = 0;
-    new_hmac_ctx = (srtp_hmac_ctx_t*)((*a)->state);
-    memset(new_hmac_ctx, 0, sizeof(srtp_hmac_ctx_t));
+    new_hmac_ctx = (HMAC_CTX*)((*a)->state);
```

Comment:
Since state is no longer a srtp_hmac_ctx_t, can we remove the openssl values from the srtp_hmac_ctx_t struct?